rpcserver.go

```diff
@@ -294,6 +294,14 @@ func (r *rpcServer) GetInfo(context.Context,
 func (r *rpcServer) MintAsset(ctx context.Context,
 	req *mintrpc.MintAssetRequest) (*mintrpc.MintAssetResponse, error) {
 
+	if req == nil {
```

Comment:
I think the gRPC library actually makes sure this cannot happen. At least we don't have this check anywhere else. This would only be a problem if we call the methods directly. Or was there ever an instance where this could be triggered from RPC/REST?

Reply:
you are right, chat gpt fooled me. I check for the subfields and I saw that there was some reflection tricks when you use the methods to get the fields instead of accessing the fields directly. But if you send a nil req, the connection fails

`rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing: dial tcp :0: connect: connection refused"`